### ISSUE TITLE
Why continue training after training with max_iter epoches?

### ISSUE THREAD

**Issue by JinYang88** [EnyanDai/GANF#2]
Great work!

I was wondering, after the following line, 30 more epoches are trained again, why introducing this training step? 
https://github.com/EnyanDai/GANF/blob/d207f7e62ade54018e17dc78b5b148f4459dd90d/train_water.py#L188

Thanks in advance!

**Comment by EnyanDai** [comment]
In the previous process, we keep tuning the Lagrange multiplier and the penalty parameter for augmented Lagrangian. Finally, we will end with good combination of parameters. Here, we'd like to fix these parameters in the loss function and ensure the parameters of the network are fully trained until divergence.
